akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala

```diff
@@ -426,6 +428,8 @@ class InteractionPatternsSpec extends ScalaTestWithActorTestKit with AnyWordSpec
     // asking someone requires a timeout if the timeout hits without response
     // the ask is failed with a TimeoutException
     implicit val timeout: Timeout = 3.seconds
+    // implicit ActorSystem in scope
+    implicit val system: ActorSystem[_] = theSystem
```

Comment:
how was this compiling without one in scope?

Reply:
it's in scope by the testkit